Formula/percona-xtrabackup.rb

```diff
@@ -76,21 +76,6 @@ class PerconaXtrabackup < Formula
     sha256 "fc9f85fc030e233142908241af7a846e60630aa7388de9a5fafb1f3a26840854"
   end
 
-  # Fix CMake install error with manpages.
-  # https://github.com/percona/percona-xtrabackup/pull/1266
-  patch do
-    url "https://github.com/percona/percona-xtrabackup/commit/1d733eade782dd9fdf8ef66b9e9cb9e00f572606.patch?full_index=1"
-    sha256 "9b38305b4e4bae23b085b3ef9cb406451fa3cc14963524e95fc1e6cbf761c7cf"
-  end
-
-  # Patch out check for Homebrew `boost`.
-  # This should not be necessary when building inside `brew`.
-  # https://github.com/Homebrew/homebrew-test-bot/pull/820
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/030f7433e89376ffcff836bb68b3903ab90f9cdc/percona-server/boost-check.patch"
-    sha256 "3223f7eebd04b471de1c21104c46b2cdec3fe7b26e13535bdcd0d7b8fd341bde"
-  end
```

Comment:
I don't think this patch is safe to remove. Does it no longer apply?

Reply:
will double check, but I believe applying the patch was failing. 

I removed so I could debug/find later bugs in the build which appear related to mysql changes which got in to mysql formula, but not mysql-client.
https://github.com/Homebrew/homebrew-core/pull/131207

Reply:
yes, as-is patch fails.  
BuildError: Failed executing: patch -g 0 -f -p1 -i /private/tmp/percona-xtrabackup--patch-20230526-39787-6cw12h/boost-check.patch

Reply:
it looks like the new format takes in to account different paths.

```
  GET_FILENAME_COMPONENT(HOMEBREW_BASE ${HOMEBREW_HOME} DIRECTORY)
```
only slightly different.

Variant is also used in mysql
https://raw.githubusercontent.com/Homebrew/formula-patches/030f7433e89376ffcff836bb68b3903ab90f9cdc/mysql/boost-check.patch

Reply:
Used newer version.